diff --git a/deps/CMakeLists.txt b/deps/CMakeLists.txt
index e23a94a..67927ca 100644
--- a/deps/CMakeLists.txt
+++ b/deps/CMakeLists.txt
@@ -53,7 +53,6 @@ if(PythonSupport OR Build_Documentation)
     GIT_REPO https://github.com/TRIQS/cpp2py
     VERSION 2.0
     GIT_TAG master
-    BUILD_ALWAYS
     EXCLUDE_FROM_ALL
   )
 endif()
@@ -62,7 +61,6 @@ endif()
 external_dependency(GTest
   GIT_REPO https://github.com/google/googletest
   GIT_TAG main
-  BUILD_ALWAYS
   EXCLUDE_FROM_ALL
 )
 
@@ -72,7 +70,6 @@ if(Build_Benchs)
   external_dependency(google-bench
     GIT_REPO https://github.com/google/benchmark
     GIT_TAG v1.5.2
-    BUILD_ALWAYS
     EXCLUDE_FROM_ALL
   )
 endif()
diff --git a/python/nda_py/CMakeLists.txt b/python/nda_py/CMakeLists.txt
index 22a24ce..f1fb1e1 100644
--- a/python/nda_py/CMakeLists.txt
+++ b/python/nda_py/CMakeLists.txt
@@ -19,6 +19,7 @@ target_compile_definitions(nda_py INTERFACE
 # Install library and headers
 install(TARGETS nda_py EXPORT ${PROJECT_NAME}-targets DESTINATION lib)
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} DESTINATION include FILES_MATCHING PATTERN "*.hpp" PATTERN "*.h")
+set(PYTHON_LIB_DEST_ROOT ${CPP2PY_PYTHON_LIB_DEST_ROOT})
 set(PYTHON_LIB_DEST ${PYTHON_LIB_DEST_ROOT}/nda_py)
 install(FILES __init__.py DESTINATION ${PYTHON_LIB_DEST})
 
diff --git a/test/c++/CMakeLists.txt b/test/c++/CMakeLists.txt
index 3d75dc2..5618113 100644
--- a/test/c++/CMakeLists.txt
+++ b/test/c++/CMakeLists.txt
@@ -14,7 +14,7 @@ foreach(test ${all_tests})
   #MESSAGE("${test_name} with option ${ARGV1}")
   get_filename_component(test_dir ${test} DIRECTORY)
   add_executable(${test_name} ${test})
-  target_link_libraries(${test_name} ${PROJECT_NAME}_c gtest_main ${PROJECT_NAME}_warnings)
+  target_link_libraries(${test_name} ${PROJECT_NAME}_c gtest gtest_main ${PROJECT_NAME}_warnings)
   #target_link_libraries(${test_name} nda_c gtest_main nda_warnings)
   target_compile_options(${test_name}  PRIVATE "${ARGV1}")
   set_property(TARGET ${test_name} PROPERTY RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${test_dir})
diff --git a/test/python/CMakeLists.txt b/test/python/CMakeLists.txt
index 51aabcf..58753ca 100644
--- a/test/python/CMakeLists.txt
+++ b/test/python/CMakeLists.txt
@@ -22,6 +22,6 @@ set(all_tests wrap_basic_test copy_move_stat_test)
 foreach(test ${all_tests})
   get_filename_component(test_name ${test} NAME_WE)
   get_filename_component(test_dir ${test} DIRECTORY)
-  add_test(NAME Py_${test_name} COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/${test_dir}/${test_name}.py WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${test_dir})
+  add_test(NAME Py_${test_name} COMMAND ${CPP2PY_PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/${test_dir}/${test_name}.py WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${test_dir})
   set_property(TEST Py_${test_name} APPEND PROPERTY ENVIRONMENT PYTHONPATH=${CMAKE_CURRENT_BINARY_DIR}/${test_dir}:${PROJECT_BINARY_DIR}/python:${h5_BINARY_DIR}/python:$ENV{PYTHONPATH} ${SANITIZER_RT_PRELOAD})
 endforeach()
