diff --git a/cmake/TRIQSConfig.cmake.in b/cmake/TRIQSConfig.cmake.in
index 2eb8b521..a0abc33d 100644
--- a/cmake/TRIQSConfig.cmake.in
+++ b/cmake/TRIQSConfig.cmake.in
@@ -80,7 +80,7 @@ add_library(triqs_py INTERFACE)
 target_link_libraries(triqs_py INTERFACE nda::nda_py)
 
 # Include the exported targets of this project
-include(@CMAKE_INSTALL_PREFIX@/@FMT_CMAKE_DIR@/fmt-targets.cmake)
+include(@fmt_DIR@/fmt-targets.cmake)
 include(@CMAKE_INSTALL_FULL_LIBDIR@/cmake/triqs/triqs-targets.cmake)
 
 message(STATUS "Found TRIQSConfig.cmake with version @PROJECT_VERSION@, hash = @TRIQS_GIT_HASH@, root = @CMAKE_INSTALL_PREFIX@")
diff --git a/deps/CMakeLists.txt b/deps/CMakeLists.txt
index f1ef146d..1e9cad94 100644
--- a/deps/CMakeLists.txt
+++ b/deps/CMakeLists.txt
@@ -64,7 +64,6 @@ endif()
 external_dependency(GTest
   GIT_REPO https://github.com/google/googletest
   GIT_TAG main
-  BUILD_ALWAYS
   EXCLUDE_FROM_ALL
 )
 
@@ -74,18 +73,15 @@ if(Build_Benchs)
   external_dependency(google-bench
     GIT_REPO https://github.com/google/benchmark
     GIT_TAG main
-    BUILD_ALWAYS
     EXCLUDE_FROM_ALL
   )
 endif()
 
 # -- fmt --
-set(FMT_INSTALL ON)
 external_dependency(fmt
   GIT_REPO https://github.com/fmtlib/fmt
   GIT_TAG 10.2.1
 )
-target_compile_options(fmt PRIVATE $<$<CXX_COMPILER_ID:IntelLLVM>:-Wno-tautological-constant-compare>)
 
 # -- itertools --
 external_dependency(itertools
