diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index f69cb10b..c6e4f620 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -51,7 +51,7 @@ foreach(test ${mpi_tests})
     target_link_libraries(${test} PRIVATE
                           ${PROJECT_NAME} ${MPI_CXX_LIBRARIES} catch2-pomerol)
 
-    foreach(np 2 4 8 16)
+    foreach(np 2 4)
         set(test_parameters
             ${MPIEXEC_NUMPROC_FLAG} ${np}
             ${MPIEXEC_PREFLAGS} ${test} ${MPIEXEC_POSTFLAGS})
