diff --git a/CMakeLists.txt b/CMakeLists.txt
index b0a472b..69f422e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -60,10 +60,10 @@ configure_package_config_file(
   "${PROJECT_BINARY_DIR}/libcommuteConfig.cmake"
   INSTALL_DESTINATION lib/cmake
 )
-install(EXPORT libcommuteTargets DESTINATION lib/cmake)
+install(EXPORT libcommuteTargets DESTINATION lib/cmake/libcommute)
 install(FILES "${PROJECT_BINARY_DIR}/libcommuteConfigVersion.cmake"
               "${PROJECT_BINARY_DIR}/libcommuteConfig.cmake"
-        DESTINATION lib/cmake)
+        DESTINATION lib/cmake/libcommute)
 
 # Install pkg-config file
 configure_file(libcommute.pc.in libcommute.pc @ONLY)
