diff --git a/gftools/eval_expression.hpp b/gftools/eval_expression.hpp
index 70dd84e..caea08c 100644
--- a/gftools/eval_expression.hpp
+++ b/gftools/eval_expression.hpp
@@ -2,6 +2,7 @@
 
 #include <utility>
 #include <array>
+#include <cstddef>
 
 //#include "tuple_tools.hpp" // only for debug
 
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 2776f17..99c4289 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -26,6 +26,6 @@ grid_object_test
 foreach (test ${tests})
     set(test_src ${test}.cpp)
     add_executable(${test} ${test_src} $<TARGET_OBJECTS:libgftools>)
-    target_link_libraries(${test} gtest_main)
+    target_link_libraries(${test} gtest gtest_main)
     add_test(${test} ${test})
 endforeach(test)
